services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -313,12 +313,13 @@ def public_dict(self) -> dict[str, Any]:
     def to_client_statics(self) -> dict[str, Any]:
         data = self._export_by_alias(
             include={
-                "APP_NAME",
-                "API_VERSION",
-                "SC_VCS_URL",
-                "SC_VCS_REF",
-                "SC_BUILD_DATE",
-                "SWARM_STACK_NAME",
+                "APP_NAME": True,
+                "API_VERSION": True,
+                "WEBSERVER_PROJECTS": {"PROJECTS_MAX_NUM_RUNNING_DYNAMIC_NODES"},
```

Comment:
MINOR: keep them alphabetic

Reply:
yep, and fixed the others that weren't as well.